### ISSUE TITLE
Error building wheel for minotaur

### ISSUE THREAD

**Issue by damianmoore** [giannitedesco/minotaur#5]
Hi there, this project looks interesting, however I'm having trouble installing. `pip install minotaur==0.0.3` generates this:
```
  Building wheel for minotaur (setup.py): started
  Building wheel for minotaur (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-zgtvfvog/minotaur_70dba2a665fc411581cc3b0700352d54/setup.py'"'"'; __file__='"'"'/tmp/pip-install-zgtvfvog/minotaur_70dba2a665fc411581cc3b0700352d54/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-pybliqpe
       cwd: /tmp/pip-install-zgtvfvog/minotaur_70dba2a665fc411581cc3b0700352d54/
  Complete output (41 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.8
  creating build/lib.linux-x86_64-3.8/minotaur
  copying minotaur/__init__.py -> build/lib.linux-x86_64-3.8/minotaur
  copying minotaur/__version__.py -> build/lib.linux-x86_64-3.8/minotaur
  copying minotaur/_minotaur.py -> build/lib.linux-x86_64-3.8/minotaur
  copying minotaur/_base.py -> build/lib.linux-x86_64-3.8/minotaur
  copying minotaur/_mask.py -> build/lib.linux-x86_64-3.8/minotaur
  copying minotaur/__main__.py -> build/lib.linux-x86_64-3.8/minotaur
  copying minotaur/_event.py -> build/lib.linux-x86_64-3.8/minotaur
  running egg_info
  writing minotaur.egg-info/PKG-INFO
  writing dependency_links to minotaur.egg-info/dependency_links.txt
  writing top-level names to minotaur.egg-info/top_level.txt
  reading manifest file 'minotaur.egg-info/SOURCES.txt'
  writing manifest file 'minotaur.egg-info/SOURCES.txt'
  copying minotaur/_inotify.c -> build/lib.linux-x86_64-3.8/minotaur
  running build_ext
  building 'minotaur._inotify' extension
  creating build/temp.linux-x86_64-3.8
  creating build/temp.linux-x86_64-3.8/minotaur
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.8 -c minotaur/_inotify.c -o build/temp.linux-x86_64-3.8/minotaur/_inotify.o
  In file included from /usr/local/include/python3.8/Python.h:137,
                   from minotaur/_inotify.c:2:
  minotaur/_inotify.c: In function ‘PyInit__inotify’:
  minotaur/_inotify.c:120:26: error: ‘IN_MASK_CREATE’ undeclared (first use in this function); did you mean ‘IN_MASK_ADD’?
    PyModule_AddIntMacro(m, IN_MASK_CREATE);
                            ^~~~~~~~~~~~~~
  /usr/local/include/python3.8/modsupport.h:139:67: note: in definition of macro ‘PyModule_AddIntMacro’
   #define PyModule_AddIntMacro(m, c) PyModule_AddIntConstant(m, #c, c)
                                                                     ^
  minotaur/_inotify.c:120:26: note: each undeclared identifier is reported only once for each function it appears in
    PyModule_AddIntMacro(m, IN_MASK_CREATE);
                            ^~~~~~~~~~~~~~
  /usr/local/include/python3.8/modsupport.h:139:67: note: in definition of macro ‘PyModule_AddIntMacro’
   #define PyModule_AddIntMacro(m, c) PyModule_AddIntConstant(m, #c, c)
                                                                     ^
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for minotaur
```

Python version: 3.8.6
Base docker image: python:3.8.6-slim-buster

**Comment by giannitedesco** [comment]
Thanks for the report. It looks like `IN_MASK_CREATE` is relatively new. I'll take a look at fixing it

**Comment by giannitedesco** [comment]
@damianmoore can you confirm the changes I just pushed to master work for you?

```bash
git clone git@github.com:giannitedesco/minotaur.git
pip install minotaur.git
```

If that all works, I will cut a new release.

**Comment by giannitedesco** [comment]
A new release is cut, which includes the fix. Please re-open if not fixed, or create a new issue if there's anything else. Thanks again for reporting this.
